src/configure/helper/templateHelper.ts

```diff
@@ -814,7 +814,7 @@ let azurePipelineTemplates: { [key in SupportedLanguage]: LocalPipelineTemplate[
 let githubWorklowTemplates: { [key in SupportedLanguage]: LocalPipelineTemplate[] } = {
     'docker': [
         {
-            label: 'Containerized application to AKS',
+            label: 'Containerized application to Azure Kubernetes Service (AKS)',
```

Comment:
Hoping this label update is made in template service as well?
As we need to have the labels exactly same.

Reply:
yes